oci-unit-tests/telegraf_test.sh

```diff
@@ -47,21 +47,21 @@ docker_run_telegraf() {
 wait_telegraf_container_ready() {
     local container="${1}"
     local log="\[agent\] Config: Interval:.+, Quiet:.+, Hostname:.+"
-    wait_container_ready "${container}" "${log}"
+    wait_container_ready "${container}" "${log}" 30 true
 }
 
 # Verify that telegraf is up and running.
 test_telegraf_up() {
     debug "Creating telegraf container"
-    container=$(docker_run_telegraf)
+    container=$(docker_run_telegraf --network host)
```

Comment:
This test seems to be depending on the host different from the other tests. I think we should document this better since this is an exception. Maybe add a comment on top of it explaining the why?

Reply:
Yeah, I put a comment in the test below this one explaining why we're using the host network, but now that this test also depends on the host, I think it's fair to replicate the comment.  That's what I did.